web/src/components/njwds-extended/PrimaryButton.tsx

```diff
@@ -3,23 +3,23 @@ import { forwardRef, ReactElement, Ref } from "react";
 
 export type PrimaryButtonColors =
   | "primary"
-  | "secondary"
-  | "accent-cool-lightest"
+  | "feedback"
   | "accent-cool-darker"
   | "accent-cooler"
-  | "info";
+  | "info"
+  | "tertiary";
```

Comment:
So I'm not sure exactly what we want precent to be for this. 

We have a SecondaryButton compoennt and QuaternaryButton component

But the primary button has a bunch of sub types which are used for specific things so I'm confused as to where the line is or should be

Reply:
[dust] When I consider the distinctions between buttons, I examine the styling applied at the parent level. For a primary button, the container itself has a solid fill, while the secondary button has a border, and an unstyled button has neither. 

The children elements of a button, including text or graphics, should not influence the decision on how many buttons to create or how to distinguish between them. For instance, in the case of the 'primary' button, the background color is determined by the 'primary button colors' property.

The available colors for the primary colors property are directly linked to the default background colors. As mentioned earlier, you can easily view them by clicking directly into the button.